quick_template/core/config.nims

```diff
@@ -0,0 +1,26 @@
+## This file can also be placed at scene root to act as the default for all workspaces
+import os
+
+--app: lib
+--noMain: on
+--define: nimPreviewDotLikeOps
+
+let workspaceName = projectDir().splitFile.name
+
+switch("define", "ExtensionName:" & workspaceName)
+switch("out", "$projectdir/lib/" & workspaceName.toDll)
+
+# begin Nimble config (version 2)
+when withDir(thisDir(), system.fileExists("nimble.paths")):
+  include "nimble.paths"
+# end Nimble config
+
+# begin gdext spacification
+when withDir(thisDir(), system.fileExists("bootstrapconf.nims")):
+  include "bootstrapconf.nims"
+# end gdext spacification
```

Comment:
This section is not necessary for this configuration.

Reply:
I had that thought, but is this an issue? It's a template, someone might end up creating that file (if it is still in spec) especially if they use multiple extensions so I don't see the point in trimming it out.